plc4j/drivers/opcua/src/test/resources/log4j.properties

```diff
@@ -20,5 +20,5 @@ appender.out.type=Console
 appender.out.name=out
 appender.out.layout.type=PatternLayout
 appender.out.layout.pattern=[%30.30t] %-30.30c{0} %-5p %m%n
-rootLogger.level=ERROR
```

Comment:
check if we set this to error again before we merge.

Reply:
I removed log4j.properties and left only logback.